.pre-commit-config.yaml

```diff
@@ -1,27 +1,34 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v2.5.0
     hooks:
-    -   id: check-json
-    -   id: check-yaml
-    -   id: end-of-file-fixer
-    -   id: trailing-whitespace
+      -   id: check-json
+      -   id: check-yaml
+      -   id: end-of-file-fixer
+      -   id: trailing-whitespace
 
--   repo: https://github.com/ambv/black
+  - repo: https://github.com/ambv/black
     rev: 20.8b1
     hooks:
-    - id: black
+      - id: black
 
--   repo: https://gitlab.com/pycqa/flake8
+
+  - repo: https://gitlab.com/pycqa/flake8
     rev: 3.7.9
     hooks:
-    - id: flake8
+      - id: flake8
+
 
--   repo: local
+  - repo: local
     hooks:
-    - id: pylint
-      name: pylint
-      language: system
-      types: [python]
-      entry: "python -m pylint"
-exclude: "tests/jobs/localisation/snapshots/"
+      - id: pylint
+        name: pylint
+        language: system
+        types: [python]
+        entry: "python -m pylint"
+exclude: tests/jobs/localisation/snapshots/
+#exclude:   |
+#        (?x)^(
+#        tests/jobs/localisation/snapshots/ |
+#        tests/legacy_test_data/poly_gen_param/
+#        )$
```

Comment:
Comments left over from debugging? Also, are the other diffs in this file needed? Does not look like they do anything?

Reply:
Yes, can reset this file back to its original before my edits in this commit. How to do that in an easy way?

Reply:
This should work: `git checkout upstream/main .pre-commit-config.yaml`

Reply:
OK (except that I have called the remote for origin instead of upstream)